packages/admin/cms-admin/src/common/XMLToState.ts

```diff
@@ -0,0 +1,152 @@
+import { DraftInlineStyleType, RawDraftContentBlock } from "draft-js";
+
+interface InlineStyle {
+    id: number;
+    offset: number;
+    length: number;
+}
+
+interface EntityRange {
+    key: number;
+    offset: number;
+    length: number;
+}
+
+export const XMLToState = (block: RawDraftContentBlock): RawDraftContentBlock => {
```

Comment:
This naming is misleading: the function doesn't handle the complete Draft.js state, but only a single block. Could this function handle the complete state? Might me useful for planned features, such as saving HTML in the database instead of Draft.js state